app/assets/stylesheets/partials/_navbar.scss

```diff
@@ -5,3 +5,16 @@
     max-height: 2em;
   }
 }
+
+.navbar-lewagon {
+  justify-content: space-between;
+  background: white;
+}
+
+.navbar-lewagon .navbar-collapse {
+  flex-grow: 0;
+}
+
+.navbar-lewagon .navbar-brand img {
+  width: 40px;
+}
```

Comment:
since you're using scss you can do:
```
.navbar-lewagon {
  justify-content: space-between;
  background: white;

  .navbar-collapse {
    flex-grow: 0;
  }
  
  .navbar-brand img {
    width: 40px;
  }
}

```

Reply:
True, i did not realize that it was there. I dont even use it. I ve erased at: beaa993